frontend/model/contracts/chatroom.js

```diff
@@ -287,7 +287,6 @@ sbp('chelonia/defineContract', {
           }
         }
 
-        state.members[memberID].leftHeight = height
```

Comment:
Was this not used for anything?

Reply:
See the line below that makes this line pointless